### PR TITLE
ONNX Runtime training cpu package name for ADO

### PR DESCRIPTION
**Pull request by baijumeswani**
So far, ONNX Runtime training only had the `onnxruntime-training` python package.

With this PR, we will also publish the `onnxruntime-training-cpu` package. The `onnxruntime-training` package will solely be for cuda.

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cpu.yml [hunk 1]
```diff
@@ -76,6 +76,8 @@ stages:
               --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
               -e NIGHTLY_BUILD \
               -e BUILD_BUILDNUMBER \
+              -e ORT_DISABLE_PYTHON_PACKAGE_LOCAL_VERSION \
+              -e DEFAULT_TRAINING_PACKAGE_DEVICE \
```
Comment (askhade) [inline tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cpu.yml:80]: Do we need both these variables? Can we keep just one? This PR is OK, we can do the refactoring later. I am just curious whether we can keep just 1.

**Review by snnn — APPROVED** [review]

**Review by askhade — APPROVED** [review]
LGTM

**Comment by baijumeswani** [comment]
Thanks for the review @snnn @askhade :)

**Pull request merged**